File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/request/RunnerInfoRequest.java
package lunalauf.rms.utilities.network.communication.message.request;

import com.google.gson.annotations.SerializedName;
import lunalauf.rms.utilities.network.communication.message.type.RequestType;

public class RunnerInfoRequest extends Request {

    @SerializedName("runner-id")
    public long runnerId;

    public RunnerInfoRequest() {
        super(RequestType.RUNNER_INFO);
    }

}

File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/request/RunnerInfoRequest.kt
package lunalauf.rms.utilities.network.communication.message.request

import com.google.gson.annotations.SerializedName
import lunalauf.rms.utilities.network.communication.message.type.RequestType

class RunnerInfoRequest : Request(RequestType.RUNNER_INFO) {
    @SerializedName("runner-id")
    var runnerId: Long = 0
}
